src/service/variable-source.js

```diff
@@ -236,11 +236,13 @@ export class VariableSource {
     keys.sort((s1, s2) => s2.length - s1.length);
     const all = keys.join('|');
     // Match the given replacement patterns, as well as optionally
-    // arguments to the replacement behind it in parantheses.
+    // arguments to the replacement behind it in parentheses.
     // Example string that match
     // FOO_BAR
     // FOO_BAR(arg1)
     // FOO_BAR(arg1,arg2)
-    return new RegExp('\\$?(' + all + ')(?:\\(([0-9a-zA-Z-_.,]+)\\))?', 'g');
+    // FOO_BAR(arg1, arg2)
+    return new RegExp('\\$?(' + all + ')' +
+        '(?:\\((\\s*(?!,)(?:\\s*,?\\s*[0-9a-zA-Z-_.]*)*)\\))?', 'g');
```

Comment:
@jridgewell Note that, compared to your suggested regex, I had to change your + to a * so that it would still handle missing params such as FOO(a,,c)

Reply:
Is that allowed?

Reply:
Yes - please see this test, and the one immediately following:
https://github.com/ampproject/amphtml/blob/2fc55d619e9f20632a327b7eed7ea52b089bea9e/test/functional/test-url-replacements.js#L363

Reply:
Ohh, in that case we can use a positive lookahead instead, which'll allow a `FOO(, test)`:

```js
/\(((?:\s*[0-9a-zA-Z-_.]*\s*(?=,|\)),?)*)\s*\)/
```

Reply:
Done.